pulsar-client-cpp/tests/ConsumerTest.cc

```diff
@@ -763,4 +763,36 @@ TEST(ConsumerTest, testPartitionsWithCloseUnblock) {
     thread.join();
 }
 
+TEST(ConsumerTest, testGetLastMessageIdBlockWhenConnectionDisconnected) {
+    int operationTimeout = 10;
```

Comment:
10 seconds might be too long for a single test, maybe you can reduce it to 5 seconds (just a suggestion).